front/src/config/store.js

```diff
@@ -4,15 +4,18 @@ import produce from 'immer';
 
 import trackApi from '../api/trackApi';
 import submissionApi from '../api/submissionApi';
+import userApi from '../api/userApi';
 
 const TOGGLE_DARK_THEME = 'TOGGLE_DARK_THEME';
 const FETCH_TRACKS = 'FETCH_TRACKS';
 const FETCH_TRACK = 'FETCH_TRACK';
 const FETCH_PROBLEMS = 'FETCH_PROBLEMS';
 const FETCH_PROBLEM = 'FETCH_PROBLEM';
 const CHANGE_SUBMISSION = 'CHANGE_SUBMISSION';
+const CHANGE_MY_PROFILE = 'CHANGE_MY_PROFILE';
 
 const initialState = {
+  profile: false,
```

Comment:
is it a boolean?

Reply:
It depends : it is either a Boolean or an object

Reply:
if user is disconnected = false else profile

Reply:
hmm it would be better to have either an empty object or a `null`

Reply:
I used null as default and/or disconnected value for profile